### ISSUE TITLE
[Bug-App]:

### ISSUE THREAD

**Issue by lovettchris** [wandb/wandb#8670]
### Describe the bug

Jobs submitted from a Linux machine include the "Command", line and OS, Python and GPU info like this:

![Image](https://github.com/user-attachments/assets/90ecaceb-2686-42df-837a-732d70f7b262)

On Windows I don't get any of that, how can I fix this so that more info is added to the job header on Windows?  I'd like to add a few more properties also (my git branch and commit id for example):

![Image](https://github.com/user-attachments/assets/53219b83-ab6a-4380-bd3c-e465b9b5f840)

**Comment by fmamberti-wandb** [comment]
Hi @lovettchris, thank you for reaching out with your report. I will work to reproduce this on our end will keep you posted on the progress.

On the Windows device, are you running the same script as on Linux? What environment manager are you on?

Regarding the git information - you can see more info [here](https://docs.wandb.ai/guides/track/tracking-faq/#how-can-i-save-the-git-commit-associated-with-my-run). For these to show, your current working directory should be the same as your git synced folder.

**Comment by lovettchris** [comment]
Yes exactly the same python script on both Linux and Windows and they are both literally running out of the same git repo also.  I got a 404 not found from your link to https://docs.wandb.ai/guides/track/tracking-faq/#how-can-i-save-the-git-commit-associated-with-my-run, but I found it here: https://docs.wandb.ai/support/save_git_commit_associated_run/.

But perhaps this is fixed already because I now see the Command and System Info for my Windows machines, but I'm not seeing the git information, but that might be because the job cwd for this windows job is not in my git repo, even though the python script that it is executing IS in my git repo - is that something you could also check for? If the command is "python" find where the script is located and if that script is in a git repo, then log that info?

![Image](https://github.com/user-attachments/assets/4b13c282-9079-4a36-bcc7-0733488633f4)

**Comment by fmamberti-wandb** [comment]
Hi @lovettchris , sorry about the broken link - we updated the docs recently and the page was moved, great that you were able to find it.

Regarding the git info - your assumption is correct. Currently, the cwd when running the python script needs to be within the git directory for the git information to be logged. 

I've logged a feature request on your behalf to detect and record git information if the script itself is in a git repository while the cwd is not.

**Comment by lovettchris** [comment]
Awesome, thanks.
